src/bokeh/models/renderers/contour_renderer.py

```diff
@@ -108,6 +104,8 @@ def construct_color_bar(self, **kwargs) -> ContourColorBar:
         as the ContourRenderer. Extra keyword arguments may be passed in to
         control ``BaseColorBar`` properties such as `title`.
         '''
+        from ..annotations import ContourColorBar
+        from ..tickers import FixedTicker
```

Comment:
Just wanted to check that putting the `import`s here is motivated by reducing the top-level import time?

Reply:
No, it's to avoid cyclic module dependencies. The first one is mandatory, otherwise bokeh is completely broken. The second one isn't strictly necessary, but given it's not being used in properties, then it doesn't hurt to put it here, to avoid problems in the future.